controllers/cloud.redhat.com/metrics.go

```diff
@@ -67,6 +67,14 @@ var (
 		},
 		[]string{"user"},
 	)
+
+	enoVersion = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "eno_version",
+			Help: "ENOVersion 1 if present, 0 if not",
```

Comment:
Is this help message true? I don't see this logic implemented.

Reply:
I copied this directly off Clowder without thinking about that. I updated to with something that makes more sense. Thanks!

Reply:
So, the logic is in the previous comment you made:
`enoVersion.With(prometheus.Labels{"version": Version}).Inc()`
This will increment if there is a version in the `version.txt` file.